modules/apps/27-interchain-accounts/types/genesis.go

```diff
@@ -80,6 +80,10 @@ func (gs ControllerGenesisState) Validate() error {
 		}
 	}
 
+	if err := gs.Params.Validate(); err != nil {
```

Comment:
these technically cannot return an error. Validate only checks that params contain a bool as its field (the compiler would break if it didn't)

Keeping here for when we add more params which do validation

Reply:
No problem, I'll likely have something ready today. I'll just need to pull in your latest changes. Ack the additions :)